src/ggsn_gn.erl

```diff
@@ -251,10 +251,10 @@ handle_event(info, _Info, _State, Data) ->
     keep_state_and_data;
 
 handle_event({timeout, context_idle}, stop_session, _state, Data) ->
-    delete_context(undefined, normal, Data);
+    delete_context(undefined, inactivity_timeout, Data);
```

Comment:
we might distinguish between U and C idle timeout.

Reply:
Yes, that's definitely a good idea. We also still need to improve handling different error scenarios from AAA. I'd propose to address those in a next PR.